### ISSUE TITLE
Feature Request:  Remove line numbering when copying from One Note to the clipboard

### ISSUE THREAD

**Issue by krasian** [elvirbrk/NoteHighlight2016#69]
When copying from One Note to the clipboard, I would like a way in which the line numbering does not get copied.  This would be helpful as I normally paste into One Note with line numbering.  When copying from One Note, I would like to copy the code without line numbering and be able to paste directly into Notepad or PowerShell ISE.

Perhaps this could be achieved by one of the following methods:

1. using tables (line number in the first column of each row and the code for the line in the second column of each line number row)
2. adding a button to the NoteHighlight ribbon bar that "Copy numbered code to clipboard" that will automatically delete the first 6 characters of each line.

**Comment by GREGG-CHRISTOFFERSON** [comment]
I agree, this should be implemented.  Or just remove the line numbering all together.  Having the line numbers there makes it impossible to copy and paste code from onenote to other text editor without having to delete all the line numbers.

**Comment by elvirbrk** [comment]
Not sure I understand the issue.
Is it problem with line numbers added by this addin or code you are pasting already has line numbers?
If you insert code in OneNote using this addin, then it's already using tables, and row numbers are in different column.

**Comment by krasian** [comment]
When pasting into the popup box, you can check 'line number' to add line numbers.  That part works great.  The question is that when you copy from OneNote (after you had originally pasted from the addin into OneNote), it is helpful to have the line numbers removed before pasting into your editor.

As I use VI and PowerShell ISE often, they don't recognize columns and rows, so when I paste into PS ISE, I have to use another editor to remove the line numbers before pasting.  VI can much easier remove the line numbers (actually, just deleting the first 6 characters from every line).
